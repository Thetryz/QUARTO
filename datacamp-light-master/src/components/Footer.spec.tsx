import * as React from "react";
import { shallow } from "enzyme";

import { Footer } from "./Footer";

describe("Footer", () => {
  it("should exist", () => {
    expect(Footer).toBeDefined();
  });

  it("should match snapshot", () => {
    const tree = shallow(<Footer />);
    expect(tree).toMatchSnapshot();
  });
});
