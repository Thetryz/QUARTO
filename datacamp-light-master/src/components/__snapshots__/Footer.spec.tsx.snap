// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<div
  className="footer"
>
  <Button
    appearance="primary"
    disabled={false}
    onClick={[Function]}
    size="small"
  >
    Run
  </Button>
  <Connect(t)
    className="status"
  />
  <div
    className="spacer"
  />
  <Connect(RestartSessionButton)
    className="restart"
    visible={false}
  />
  <a
    className="logo"
    href="https://www.datacamp.com/?utm_source=datacamp_light&utm_campaign=powered_by_datacamp"
    target="_blank"
    title="Powered by DataCamp"
  >
    <test-file-stub />
  </a>
</div>
`;
